pcdsdevices/lxe.py

```diff
@@ -272,6 +274,8 @@ class LaserTiming(FltMvInterface, PVPositioner):
     user_offset = Cpt(AttributeSignal, attr='_user_offset',
                       kind='normal',
                       doc='A Python-level user offset.')
+    # Current Dial position
+    dial_position = Cpt(EpicsSignalRO, '.DRBV', kind='normal')
```

Comment:
I'm trying to figure out what this should be- I don't think there's actually a `.DRBV` field on the LXT prefix.

Reply:
That would make sense since I got `None` for that - thanks for pointing this out, i'm going to look at the old code and see what they used

Reply:
I can't figure out what they're using, it seems like in the old code the had a function `wm_dial()` that they kept using for the dial position, but that one was still using the `.DRBV` field.... I am still looking into this one

Reply:
I used the `_fs_tgt_time` as the actual `dial` position here. The `user` position gets calculated in the `_ScaledUnitConversionDerivedSignal` from what I understand, so the `setpoint` is the actual `user` position